static/app/components/profiling/breadcrumb.tsx

```diff
@@ -62,13 +62,13 @@ function trailToCrumb(
     }
     case 'flamegraph': {
       return {
-        to: flamegraphRouteWithQuery({
+        to: generateFlamegraphRouteWithQuery({
           location,
           orgSlug: organization.slug,
           projectSlug: trail.payload.projectSlug,
           profileId: trail.payload.profileId,
         }),
-        label: trail.payload.interactionName,
+        label: t('Flamegraph'),
```

Comment:
Any reason for this? This is the only place on the flamegraph that contextualized what we're looking at.